bool solar_charge_status
---
std_msgs/String result